code/parseParams.m

```diff
@@ -34,7 +36,17 @@
 addParameter(p,'date',def_date,validDate);
 parse(p,Data,varargin{:});
 params = p.Results;
-params.p1 = p.Results.dT*p.Results.num_lags;
+params.lambda = stringcheck(params.lambda);
+params.dT = stringcheck(params.dT);
+params.num_lags = stringcheck(params.num_lags);
+params.replicate = stringcheck(params.replicate);
+params.prob_zero_removal = stringcheck(params.prob_zero_removal);
+params.prob_remove_samples = stringcheck(params.prob_remove_samples);
+params.kernel_width = stringcheck(params.kernel_width);
+params.ID = stringcheck(params.ID);
+
+params.lambda = sort(params.lambda,'descend');
```

Comment:
Want to provide glmnet the largest lambda first now that we use the warm start for the solver.